css/owned-list-pages.styl

```diff
@@ -19,6 +24,16 @@
         margin: 0
         padding: 2.33em 0
 
+  .in-project-context
+
+    .collection-results-counter
+
+        p:first-child
+          padding-top: 0px !important
+
+     nav.pagination
+        color: #404040
```

Comment:
Jordan is gradually introducing new colours, I didn't want to change it everywhere yet, but maybe I will. I'll add a constant anyway. Thanks.